--- FAIL: TestSchemaString (0.00s)
    --- FAIL: TestSchemaString/hasInverse-with-type-having-directive.graphql (0.00s)
        schemagen_test.go:84: 
            	Error Trace:	schemagen_test.go:84
            	Error:      	Received unexpected error:
            	            	input:38: Cannot redeclare directive auth.
            	Test:       	TestSchemaString/hasInverse-with-type-having-directive.graphql
